### ISSUE TITLE
Log output to disk?

### ISSUE THREAD

**Issue by dbhaigh** [Azure/aztfexport#330]
TL/DR: Can we log to disk as well as to screen?  If so?  How?  If not? When?

I'd like to be able to save the output of the initial aztfy run and the subsequent import run to file (csv or tsv please) so I can more easily deal with any snags encountered

I'd rather not have to go through each screen of the output copying each page one-at-a-time ( not that annoying when you are dealing with a resource group with a small amount of resources in it - but when you are talking several hundred resources identified in a resource group?  That can get pretty old really fast)

If I'm missing something obvious?  Please let me know as it will make my life easier - if this functionality isn't available in the latest build?  Or not in the next release?  Might it please be added?  Especially for the output of ""w" import" -trying to deal with those snags is a lot of fun - as you need to access each error individually - and if you have a lot of snags to deal with?  It makes for a LOT of unnecessary grunt work

Thanks for considering this

-=A=-

.

**Comment by magodo** [comment]
@dbhaigh Are you talking about batch editing the import list and then import them non-interactively? If so, you can firstly [generate the resource mapping file](https://github.com/Azure/aztfy#terrafy-a-predefined-set-of-resources:~:text=You%20can%20generate,have%20been%20imported.), review and edit it as you wish. Then run [`aztfy mapping-file`](https://github.com/Azure/aztfy#terrafy-a-predefined-set-of-resources) subcommand by feeding that mapping file.

The title of the issue seems to ask for log output to disk, this is supported by `--log-path` option.

**Comment by dbhaigh** [comment]
I am after logging the raw output to file, yes - not that interested in batch-editing the import list - not atr this moment in time at least

--log-path option?

That doesn't appear to exist in v0.9.0 - will try cloning the repo, compiling it locally, and trying again

**Comment by dbhaigh** [comment]
Looks like a local compile has added both the --log-level and --log-path options (and a hew other new CLI options too)

I'll run these new options through their paces and see if it's generating the information I'm after - will update later

**Comment by magodo** [comment]
@dbhaigh Since we haven't get further response from you, I'll mark this issue as closed for now. Feel free to reopen it if the issue still remains.
